### PR TITLE
Ddcnl-9217: toggle and logs

### PR DESCRIPTION
**Pull request by pascal22p**
- add logging of expiration date of certs
- add toggle to switch between 2 sets of certs

### CODE REVIEW

File: app/controllers/ApplePassController.scala [hunk 1]
```diff
@@ -44,13 +45,14 @@ class ApplePassController @Inject()(
...
-      })
+      passService.createPass(passRequest.fullName, passRequest.nino).fold(
+        error =>
+          InternalServerError(Json.obj(
+            "status" -> "500",
+            "message" -> error.getMessage
+          )),
+          result => Ok(result)
+      )
     }
```
Comment (SandeepUmredkar) [inline app/controllers/ApplePassController.scala:56]: Few Unused imports in the file. Future and InternalServerError

File: app/services/ApplePassService.scala [hunk 2]
```diff
@@ -16,10 +16,12 @@
 
 package services
 
+import cats.data.EitherT
 import config.AppConfig
 import models.apple.ApplePassCard
 import play.api.Logging
 import repositories.ApplePassRepoTrait
+import uk.gov.hmrc.mongoFeatureToggles.services.FeatureFlagService
```
Comment (SandeepUmredkar) [inline app/services/ApplePassService.scala:24]: UnUsed

File: app/services/SignatureService.scala [hunk 3]
```diff
@@ -20,14 +20,16 @@ import com.nimbusds.jose.util.X509CertUtils
 import org.bouncycastle.asn1.cms.{AttributeTable, CMSAttributes}
 import org.bouncycastle.asn1.x509.Attribute
 import org.bouncycastle.asn1.{ASN1EncodableVector, DERSet, DERUTCTime}
+import org.bouncycastle.cert.X509CertificateHolder
```
Comment (SandeepUmredkar) [inline app/services/SignatureService.scala:23]: UnUsed

File: app/controllers/ApplePassController.scala (hunk 1 above)
Comment (pascal22p) [reply app/controllers/ApplePassController.scala:55]: done

File: app/services/ApplePassService.scala (hunk 2 above)
Comment (pascal22p) [reply app/services/ApplePassService.scala]: done

File: app/services/SignatureService.scala (hunk 3 above)
Comment (pascal22p) [reply app/services/SignatureService.scala]: done

**Review by SandeepUmredkar — APPROVED** [review]

**Pull request closed without merging**